Sources/TuistLoader/Loaders/ManifestModelConverter.swift

```diff
@@ -8,5 +8,9 @@ import TuistSupport
 /// A component responsible for converting Manifests (`ProjectDescription`) to Models (`TuistCore`)
 public protocol ManifestModelConverting {
```

Comment:
Nitpick: If this file only contains the protocol, I'd name the file `ManifestModelConverting`.

Reply:
this is fixed [here](https://github.com/tuist/tuist/pull/2798/files#diff-ef28cdd3469f650835405e3926a63ec4f37d73c5b7c6faa8347586579ea2bd9cR13)